dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers.ts

```diff
@@ -1,22 +1,81 @@
-import { FeeRecordEntity, UtilisationReportEntity } from '@ukef/dtfs2-common';
+import {
+  CurrencyAndAmount,
+  FeeRecordEntity,
+  UtilisationReportEntity,
+  divideAmountByExchangeRate,
+} from '@ukef/dtfs2-common';
 import { FeeRecordItem, UtilisationReportReconciliationDetails } from '../../../../types/utilisation-reports';
 import { getBankNameById } from '../../../../repositories/banks-repo';
 import { NotFoundError } from '../../../../errors';
 
 /**
- * Maps the fee record entity to the reconciliation details fee record item
+ * Maps the fee record entity to the reported fees
  * @param feeRecord - The fee record entity
- * @returns The mapped fee record
+ * @returns The reported fees
  */
-export const mapFeeRecordEntityToReconciliationDetailsFeeRecordItem = (feeRecord: FeeRecordEntity): FeeRecordItem => ({
-  facilityId: feeRecord.facilityId,
+const mapFeeRecordEntityToReportedFees = (feeRecord: FeeRecordEntity): CurrencyAndAmount => ({
+  currency: feeRecord.feesPaidToUkefForThePeriodCurrency,
+  amount: feeRecord.feesPaidToUkefForThePeriod,
 });
 
 /**
- * Maps the utilisation report entity to the reconciliation item
- * @param utilisationReport - The utilisation report
- * @returns The summary item
- * @throws {Error} If a bank cannot be found with the matching bank id
+ * Maps the fee record entity to the reported payments
+ * @param feeRecord - The fee record entity
+ * @returns The reported fees
+ */
+const mapFeeRecordEntityToReportedPayments = (feeRecord: FeeRecordEntity): CurrencyAndAmount => {
+  const { paymentCurrency, feesPaidToUkefForThePeriodCurrency, feesPaidToUkefForThePeriod, paymentExchangeRate } =
+    feeRecord;
+
+  if (paymentCurrency === feesPaidToUkefForThePeriodCurrency) {
+    return {
+      currency: paymentCurrency,
+      amount: feesPaidToUkefForThePeriod,
+    };
+  }
+
+  const feesPaidToUkefForThePeriodInPaymentCurrency = divideAmountByExchangeRate(
+    feesPaidToUkefForThePeriod,
+    paymentExchangeRate,
+    2,
+  );
+
+  return {
+    amount: feesPaidToUkefForThePeriodInPaymentCurrency,
+    currency: paymentCurrency,
+  };
+};
+
+/**
+ * Maps a fee record entity to a fee record item
+ * @param feeRecord - The fee record entity
+ * @returns The mapped fee record
+ */
+export const mapFeeRecordEntityToReconciliationDetailsFeeRecordItem = (feeRecord: FeeRecordEntity): FeeRecordItem => {
+  const reportedFees = mapFeeRecordEntityToReportedFees(feeRecord);
+  const reportedPayments = mapFeeRecordEntityToReportedPayments(feeRecord);
+
+  const totalReportedPayments = reportedPayments;
+
+  return {
+    id: feeRecord.id,
+    facilityId: feeRecord.facilityId,
+    exporter: feeRecord.exporter,
+    reportedFees,
+    reportedPayments,
+    totalReportedPayments,
+    paymentsReceived: null,
+    totalPaymentsReceived: null,
+    status: 'TO_DO',
```

Comment:
Should we actually be adding a database column for this, populating on creation and returning the value as part of this PR?